bin/disco_vpc_ui.py

```diff
@@ -28,6 +28,9 @@ def parse_arguments():
                                help='What to call the new environment.')
     parser_create.add_argument('--type', dest='vpc_type', required=True,
                                help='What type of environment to create (as defined in config).')
+    parser_create.add_argument('--skip-enis', dest='skip_enis', action='store_const',
```

Comment:
Why would we want to stop this behavior? Sounds like we're fixing a race condition / bug here, no?